scripts/install-cni.sh

```diff
@@ -329,13 +329,16 @@ MTU_SOURCE="<default>"
 # last field, which is the interface name. We stick to using grep to avoid
 # introducing too many new dependencies.
 
-default_nic=$(route -n | grep -E '^0\.0\.0\.0\s+\S+\s+0\.0\.0\.0' | grep -oE '\S+$')
+# In the case where there are multiple default routes (e.g. multi-networking
+# with route manager), try each one starting from the lowest metric value.
+readarray -t default_nics < <(route -n | grep -E '^0\.0\.0\.0\s+\S+\s+0\.0\.0\.0' | sort -n -k5,5 | grep -oE '\S+$')
```

Comment:
not sure if there is consensus in the community, but I consider `awk` the column getter:

`... | awk 'NF{ print $NF }'`

Reply:
There's no awk available in the image.

Reply:
😭